config.go

```diff
@@ -58,12 +62,13 @@ type ConfigFilter struct {
 
 // ConfigOutput specifies the configuration for the output component.
 type ConfigOutput struct {
-	Name          string
+	Name string
+	// Procs defines the number of baker outputs running concurrently.
+	// Only set Procs to a value greater than 1 if the output is concurrent safe.
 	Procs         int
-	ChanSize      int
-	RawChanSize   int
-	Sharding      string
-	Fields        []string
+	ChanSize      int      // ChanSize represents the size of the channel to send records to the ouput component(s), the default value is 16384
+	Sharding      string   // field name to be used for sharding
+	Fields        []string // fields to provide to the output as values
```

Comment:
Not making a true github suggestion since this would probably fail on ci because of gofmt
```
	Sharding      string   // Sharding is the name of the field used for sharding
	Fields        []string // Fields holds the name of the record fields the output receives
```